### PR TITLE
Add button to candidates page

### PR DESCRIPTION
**Pull request by JamBai2812**

### CODE REVIEW

File: pages/candidates.tsx [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import {CandidateList} from "../components/CandidateList/CandidateList";
+import Link from "next/link";
 
 const Candidates: NextPage = () => {
 
     return (
         <Layout>
             <h1 className={textStyle.pageHeader}>Candidate Manager</h1>
+            <button className={scss.desktopButton}><Link href={"/create"}><p className={scss.link}>+ Create Candidate</p></Link></button>
+            <button className={scss.mobileButton}><Link href={"/create"}><p className={scss.link}>+</p></Link></button>
```
Comment (MichaelWalker) [inline pages/candidates.tsx:15]: Couple of things

1) this 'button' is really just a link - no need to wrap it in a 'button' - just style the link to look how you like.
2) in Next, `Link` elements are meant to just be wrappers around a normal `a` tag. That way, without JS they work just like normal, and when you add JS then you get pre-fetching and other nice things. - https://nextjs.org/docs/api-reference/next/link

Could you get rid of the button, and place an `a` tag inside the link?

File: pages/candidates.tsx [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 import Layout from "../components/Layout/layout";
 import {CandidateList} from "../components/CandidateList/CandidateList";
+import Link from "next/link";
 
 const Candidates: NextPage = () => {
 
     return (
         <Layout>
             <h1 className={textStyle.pageHeader}>Candidate Manager</h1>
+            <button className={scss.desktopButton}><Link href={"/create"}><p className={scss.link}>+ Create Candidate</p></Link></button>
```
Comment (MichaelWalker) [inline pages/candidates.tsx:14]: I recon we might be able to do this without having 2 buttons...
How about
```tsx
<Link href="/create">
    <a>+<span> Create Candidate</span></a>
</Link>
```

Then we can just style the `Create Candidate` label to only appear on desktop?

File: pageStyles/candidates.module.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
+﻿@import 'pageStyles/constants';
+
+.link {
+  margin: unset;
+}
+
+.desktopButton {
```
Comment (MichaelWalker) [inline pageStyles/candidates.module.scss:7]: There's quite a lot in common between these 2 - if we don't turn it into 1 button, we should try to commonise these a little.

File: pageStyles/candidates.module.scss [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  background-color: $softwireYellow;
+  border: none;
+  padding: 5px 15px;
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 10px;
+  position: fixed;
```
Comment (MichaelWalker) [inline pageStyles/candidates.module.scss:14]: This looks a little suspicious - do we definitely want it to be `fixed`? From the designs I'd have thought we'd want this to scroll with the rest of the page?

File: pageStyles/candidates.module.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  display: none;
+  
+  :hover{
+    text-decoration: underline;
+    cursor: pointer;
+  }
+    
+  @media (min-width: 768px) {
+    right: 10px;
+    top: 265px;
```
Comment (MichaelWalker) [inline pageStyles/candidates.module.scss:24]: this seems large and fragile! 😂

I'm guessing this is measured from the top of the page right now?
Looks from the design like we want it to be positioned next to the 'Select a candidate' title / in that title section.

So I'd try writing the CSS to position it based on where that is - eg absolutely position it based on that as a parent?
This should
- make it less fragile: at the moment if we change anything then we'd need to reposition this button
- make it easier to understand - much clearer where you are trying to place it.

File: pageStyles/candidates.module.scss [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    display: block;
+  }
+}
+
+.mobileButton {
+  background-color: $softwireYellow;
+  padding: 1px 8px 0 7px;
```
Comment (MichaelWalker) [inline pageStyles/candidates.module.scss:31]: this looks odd too.. I'd expect it to be symmetrical?

File: pages/candidates.tsx (hunk 1 above)
Comment (JamBai2812) [reply pages/candidates.tsx:15]: Yep replaced the button

File: pages/candidates.tsx (hunk 2 above)
Comment (JamBai2812) [reply pages/candidates.tsx]: Just one now!

File: pageStyles/candidates.module.scss (hunk 3 above)
Comment (JamBai2812) [reply pageStyles/candidates.module.scss]: One button now

File: pageStyles/candidates.module.scss (hunk 4 above)
Comment (JamBai2812) [reply pageStyles/candidates.module.scss]: Now not fixed

File: pageStyles/candidates.module.scss (hunk 5 above)
Comment (JamBai2812) [reply pageStyles/candidates.module.scss]: Hopefully less fragile now, positioned absolute relative to parent

File: pageStyles/candidates.module.scss (hunk 6 above)
Comment (JamBai2812) [reply pageStyles/candidates.module.scss]: sorted this too

**Pull request merged**